eurocodedesign/geometry/steelsections/__init__.py

```diff
@@ -181,7 +182,7 @@ def _is_valid_type(section_name: str) -> bool:
     for section_type in _SECTION_DATA.keys():
         if section_type in section_name:
             filepath = _get_data_path() \
-                       / _SECTION_DATA[section_type]["filename"]
+                       / str(_SECTION_DATA[section_type]["filename"])
```

Comment:
Do you have to do this because it is not explicity stated anywhere that this is a string?